code/modules/organs/blood.dm

```diff
@@ -119,9 +119,9 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 				death()
 
 		// Without enough blood you slowly go hungry and thirsty
-		if(blood_volume < BLOOD_VOLUME_SAFE)
-			adjustNutritionLoss(get_nutrition_mul(1,20))
-			adjustHydrationLoss(get_hydration_mul(1,10))
+		if(blood_volume <= BLOOD_VOLUME_SAFE)
+			adjustNutritionLoss(get_nutrition_mul(1,13))
```

Comment:
I would make more sense to loose the nutrition loss or switch the values of the hydration and nutrition loss.